src/models/bundle-descriptor-constraints.ts

```diff
@@ -242,6 +242,10 @@ export const MICROSERVICE_CONSTRAINTS: ObjectConstraints<Microservice> = {
     required: true,
     type: 'string'
   },
+  healthCheckIngress: {
+    required: true,
```

Comment:
Should the field healthCheckIngress be required? I think no for backwards compatibility.
In this case, in old bundles, any commands you execute returns the following error. 
```
Error: entando.json is not valid.
Field "healthCheckIngress" is required
```

Reply:
@antromeo I checked the ticket and it is not defined if the property should be required or not. So, we can update that!

Reply:
done